docs/source/models/lasers.rst

```diff
@@ -0,0 +1,218 @@
+    e^{-\imath \frac{1}{2}GDD_\mathrm{foc}(\Omega-\Omega_0)^2}
+    e^{-\imath \frac{1}{6}TOD_\mathrm{foc}(\Omega-\Omega_0)^3}\\
+  z_{\mathrm R,s} = &\ \frac{\Omega_0 w_{0,s}^2}{2c} \\
+  w_s(z) = &\ w_{0,s} \sqrt{1 + \frac{z^2}{z_{\mathrm R,s}^2}} \\
+  R_s^{-1}(z) = &\ \frac{z}{z^2 + z_{\mathrm R,s}^2} \\
+  \alpha(\Omega) = &\
+    \frac{w_0}{c}\left[
+      \Omega_0 \theta^\prime (\Omega-\Omega_0)
+      + \frac{1}{2}\left( 2 \theta^\prime + \Omega_0 \theta^{\prime\prime} \right) (\Omega-\Omega_0)^2
+      \right. \\
+    &\ \qquad \left. + \frac{1}{6}\left(
+        3\theta^{\prime\prime} + \Omega_0\theta^{\prime\prime\prime} - \Omega_0 {\theta^\prime}^3
+      \right) (\Omega-\Omega_0)^3
+    \right]
+
+Note, :math:`k_x \approx - \alpha / w_{0,x}`.
+
+The sought result for :math:`\int \hat K_y \epsilon_{y^\prime}\,\mathrm dy^\prime` is obtained from the 2D solution :math:`\hat E_x(x,z,\Omega)` by dropping :math:`\epsilon_\Omega` and :math:`\mathrm e^{-\imath(\Omega/c)z}`, plus letting :math:`\alpha \rightarrow 0` and :math:`x_0 \rightarrow 0`.
+Hence,
+
+.. math::
+  \hat E_y(y,z,\Omega)& := \int\limits_{-\infty}^{\infty}
+        \hat K_y(y^\prime, z, \Omega) \epsilon_{y^\prime} \,\mathrm dy^\prime \\
+  & = \sqrt{\frac{\Omega}{2\pi c}} \frac{e^{\imath\frac{\pi}{4}}}{\sqrt{z}}
+      \int\limits_{-\infty}^{\infty} e^{-\frac{{y^\prime}^2}{w_{0,y^\prime}^2}}
+        e^{-\imath \frac{\Omega}{2 c z} (y-y^\prime)^{2}} \,\mathrm dy^\prime \\
+  & = \left[ 1 + \frac{z^2}{z_{\mathrm R,y}^2} \right]^{-1/4}
+    \mathrm e^{-y^2\left[\frac{1}{w_y(z)^2} + \imath \frac{\Omega}{2c}R_y^{-1}(z)\right]}
+    \mathrm e^{\imath\frac{1}{2}\arctan\frac{z}{z_{\mathrm R,y}}}\,.
+
+
+Result
+^^^^^^
+According to the definitions above
+
+.. math::
+  \hat E(x,y,z,\Omega) = \hat E_x \cdot \hat E_y\,.
+
+
+3D field in time-space domain
+-----------------------------
+Derivation
+^^^^^^^^^^
+For the Fourier transform to time-space domain, again the results of [Steiniger2024]_ can be reused since the modifications due to the presence of :math:`\hat E_y` are easy to incorporate.
+In fact, when applying the transformation :math:`\Omega \rightarrow \tfrac{1}{\tau_0}\Omega^\prime + \Omega_0`, as is done in the reference,
+
+.. math::
+  \hat E_y(y,z,\frac{1}{\tau_0}\Omega^\prime + \Omega_0) =
+    \left[ 1 + \frac{z^2}{z_{\mathrm R,y}^2} \right]^{-1/4}
+    \mathrm e^{-\frac{y^2}{w_y(z)^2}} \mathrm e^{-\imath \Omega_0 \frac{y^2 R^{-1}_y(z)}{2c}}
+    \mathrm e^{\imath\frac{1}{2}\arctan\frac{z}{z_{\mathrm R,y}}}
+    \mathrm e^{-\imath \frac{y^2 R_y^{-1}(z)}{2c}\frac{1}{\tau_0}\Omega^\prime}
+
+only the last exponential depends on frequency :math:`\Omega^\prime` and needs to be taken into account in the Fourier transform.
+This last exponential simply contributes a term to :math:`\gamma_4`, which is located between eqs. (13) and (14) in [Steiniger2024]_, in the form of
+
+.. math::
+  \gamma_4\; +\negthickspace= -\frac{y^2 R_y^{-1}(z)}{2c}\frac{1}{\tau_0}\,.
```

Comment:
This renders as $+=$ - what is this suppose to mean in this mathematical context?